frontend/src/components/Buttons/SegmentedButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { SegemnetdButton } from './SegmentedButton.tsx';
+
+/**
+ * Group of buttons with same border
+ * Check out https://m3.material.io/components/segmented-buttons/overview
+ * for more information about segmented buttons
+ */
+
+interface SegmentedButtonsProps {
+    names: Array<string>;
+    onClicks: Array<React.MouseEventHandler<HTMLButtonElement> | undefined>;
```

Comment:
Instead of an array of strings and an array of onclicks, this should be a single array of [string, eventhandler]. These types allow passing different sizes which is bad, but also it's easy to mix up labels and handlers.